lib/templates/middleAssignment/boxes.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:hive/hive.dart';
+
```

Comment:
<img width="644" alt="Screenshot 2024-06-24 at 13 25 30" src="https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/02d969f7-7d9c-41af-a455-2ef61051c109">

Reply:
![image](https://github.com/timur-harin/Sum24FlutterCourse/assets/158128971/68ca89ef-a542-49eb-a9d7-127f5a98e988)

I don't have this error

Reply:
And It works fine

Reply:
I got it while running